sim_rover_1:
  # Publish joint states -----------------------------------
  joint_states:
    type: joint_state_controller/JointStateController
    publish_rate: 1000 

  # LEG effort Controllers ---------------------------------------
  bleg_r_eff:
    type: effort_controllers/JointEffortController
    joint: j_bleg_r

  bleg_l_eff:
    type: effort_controllers/JointEffortController
    joint: j_bleg_l

  fleg_r_eff:
    type: effort_controllers/JointEffortController
    joint: j_fleg_r

  fleg_l_eff:
    type: effort_controllers/JointEffortController
    joint: j_fleg_l

  # WHEEL effort Controllers ---------------------------------------
  bwheel_r_eff:
    type: effort_controllers/JointEffortController
    joint: j_bwheel_r

  bwheel_l_eff:
    type: effort_controllers/JointEffortController
    joint: j_bwheel_l

  fwheel_r_eff:
    type: effort_controllers/JointEffortController
    joint: j_fwheel_r

  fwheel_l_eff:
    type: effort_controllers/JointEffortController
    joint: j_fwheel_l

  # STEERING effort Controllers ---------------------------------------
  bsteer_r_eff:
    type: effort_controllers/JointEffortController
    joint: j_bsteer_r

  bsteer_l_eff:
    type: effort_controllers/JointEffortController
    joint: j_bsteer_l

  fsteer_r_eff:
    type: effort_controllers/JointEffortController
    joint: j_fsteer_r

  fsteer_l_eff:
    type: effort_controllers/JointEffortController
    joint: j_fsteer_l

